packages/react-native-editor/src/index.js

```diff
@@ -1,161 +1,83 @@
 /**
  * External dependencies
  */
-import { I18nManager } from 'react-native';
+import 'react-native-gesture-handler';
+import { AppRegistry } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { applyFilters, doAction } from '@wordpress/hooks';
+import { Component, cloneElement } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import './globals';
-import { getTranslation } from '../i18n-cache';
+import { omit } from 'lodash';
```

Comment:
This is currently place within the `Internal dependencies` group, but should likely be in the `External dependencies` group.

Reply:
I addressed this in https://github.com/WordPress/gutenberg/pull/37073/commits/1f7cd1eac817c078f22c0d8c84b0a1b33716be50.